tendrl/commons/tests/flows/expand_cluster/test_ec_init.py

```diff
@@ -1,4 +1,4 @@
-import __builtin__
+import builtins
```

Comment:
use builtins instead of __builtin__ in code also. Check line 50

Reply:
Done